app/src/main/res/drawable/custom_button_shape.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <gradient
+        android:startColor="#ffffff"
+        android:endColor="#ffffff"
+        android:angle="45" />
```

Comment:
Having a gradient that changes from white to white is a bit useless ahah you could have used `<solid>` instead ! But having a light gradient on the button could actually be a nice idea.

Reply:
That's true, I'm still tweaking this to have a nice effect but it is quite difficult to have something good. I'll change it to a `<solid>` before merging if I do not use the gradient. Thanks for the review!